src/librustdoc/json/conversions.rs

```diff
@@ -207,18 +208,28 @@ impl FromWithTcx<clean::TypeBindingKind> for TypeBindingKind {
 /// It generates an ID as follows:
 ///
 /// `CRATE_ID:ITEM_ID[:NAME_ID]` (if there is no name, NAME_ID is not generated).
```

Comment:
Could this doc comment be updated to the new format (and also document what the `extra` and `name` params are for